app/addClothe.tsx

```diff
@@ -1,59 +1,242 @@
+                <Text style={styles.filterText}>Tags</Text>
+                <View style={styles.pickerBox}>
 
-            <TextInput style={styles.input} placeholder="Clothe description"
-                       onChangeText={(text) => clothe.description = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+                    <Picker style={styles.picker} itemStyle={styles.pick}
+                        selectedValue={selectedType} 
+                        onValueChange={(itemValue: Clothe.Type) => 
+                            setSelectedType(itemValue) // Update the selectedType state 
+                    }>
+                        <Picker.Item label="Type" value={Clothe.Type.Unknown} />
+                        <Picker.Item label="Shirt" value={Clothe.Type.Shirt} />
+                        <Picker.Item label="Pants" value={Clothe.Type.Pants} />
+                        <Picker.Item label="Shoes" value={Clothe.Type.Shoes} />
+                    </Picker>
 
-            <TextInput style={styles.input} placeholder="Clothe image link"
-                       onChangeText={(text) => clothe.link = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+                    <Picker style={styles.picker} itemStyle={styles.pick}
+                        selectedValue={selectedColor}
+                        onValueChange={(itemValue) => 
+                            setSelectedColor(itemValue)
+                        }>
+                        <Picker.Item label="Color" value="Unknown" />
+                        <Picker.Item label="Black" value="Black" />
+                        <Picker.Item label="White" value="White" />
+                        <Picker.Item label="Gray" value="Gray" />
+                        <Picker.Item label="Navy" value="Navy" />
+                        <Picker.Item label="Blue" value="Blue" />
+                        <Picker.Item label="Green" value="Green" />
+                        <Picker.Item label="Red" value="Red" />
+                        <Picker.Item label="Yellow" value="Yellow" />
+                        <Picker.Item label="Orange" value="Orange" />
+                        <Picker.Item label="Red" value="Red" />
+                        <Picker.Item label="Pink" value="Pink" />
+                        <Picker.Item label="Purple" value="Purple" />
+                        <Picker.Item label="Brown" value="Brown" />
+                    </Picker>
 
-            <Button title="Add Clothe" onPress={addClothe}/>
+                    <Picker style={styles.picker} itemStyle={styles.pick}
+                        selectedValue={selectedSize}
+                        onValueChange={(itemValue, itemIndex) => 
+                            setSelectedSize(itemValue)
+                        }>
+                        <Picker.Item label="Sleeve Size" value={-1} />                            
```

Comment:
Value cannot be negative here, it causes a crash. Either change to non-negative or handle picker choice logic before adding clothe. See previous comment.

Reply:
I will set this to 0 as the default. It'll be good for shoes that have no sleeve as well.

Reply:
I updated this to go from 0 = Unknown, 1 = No Sleeve, 2 = Small Sleeve, 3 = Large Sleeve